From 11acc3c6aae267d603f3669ff9c46c79da4d237b Mon Sep 17 00:00:00 2001
From: Marian Chereji <marian.chereji@nxp.com>
Date: Tue, 4 Oct 2016 10:27:45 +0300
Subject: [PATCH 097/388] dpa_offload: Align number of network interfaces in
 LS1043ARDB device tree files

Upgraded the LS1043ARDB device tree files to always provide the
same number of network interfaces in all offloading example
scenarios. This helps simplify the auto validation scripts.

Signed-off-by: Marian Chereji <marian.chereji@nxp.com>
[Original patch taken from SDK-V2.0-1703]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 drivers/staging/fsl_dpa_offload/dts/fsl-ls1043a-rdb-usdpaa.dts | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/drivers/staging/fsl_dpa_offload/dts/fsl-ls1043a-rdb-usdpaa.dts b/drivers/staging/fsl_dpa_offload/dts/fsl-ls1043a-rdb-usdpaa.dts
index 29f3d1e..f64b024 100644
--- a/drivers/staging/fsl_dpa_offload/dts/fsl-ls1043a-rdb-usdpaa.dts
+++ b/drivers/staging/fsl_dpa_offload/dts/fsl-ls1043a-rdb-usdpaa.dts
@@ -105,6 +105,13 @@
 			fsl,qman-frame-queues-tx = <5008 8>;
 			local-mac-address = [00 11 22 33 44 66];
 		};
+		ethernet@12 {
+			compatible = "fsl,p4080-dpa-ethernet-macless", "fsl,dpa-ethernet-macless";
+			fsl,bman-buffer-pools = <&bp16>;
+			fsl,qman-frame-queues-rx = <7000 8>;
+			fsl,qman-frame-queues-tx = <7008 8>;
+			local-mac-address = [00 11 22 33 44 77];
+		};
 
 		dpa-fman0-oh@2 {
 			compatible = "fsl,dpa-oh";
-- 
2.9.3

